frontend/src/main/java/puf/frisbee/frontend/network/SocketRequest.java

```diff
@@ -0,0 +1,32 @@
+package puf.frisbee.frontend.network;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(as = SocketRequest.class)
+public class SocketRequest {
+    private SocketRequestType requestType;
+    private String value;
```

Comment:
Hier ist erstmal nur String, für den Frisbee müssen wir uns was überlegen - entweder eine extra Property (die dann bei den anderen Actions leer ist) oder wir packen x und y, mit Komma getrennt in den String und splitten den danach dann wieder... beides irgendwie hässlich, aber möglich?